cmd/cert-checker/main.go

```diff
@@ -441,7 +448,8 @@ type Config struct {
 		DB cmd.DBConfig
 		cmd.HostnamePolicyConfig
 
-		Workers        int `validate:"required,min=1"`
+		Workers int `validate:"required,min=1"`
+		// Deprecated: this is ignored, and cert checker always checks both expired and unexpired.
 		UnexpiredOnly  bool
```

Comment:
if you feel like it: move this field to the very bottom of the struct, since order doesn't matter, and deprecated things can be shuffled out of the way.

Reply:
Additionally, remove `"unexpiredOnly": true"` from `test/config-next/cert-checker.json`.